src/custom-elements/src/components/icon/readme.md

```diff
@@ -0,0 +1,19 @@
+# chi-icon
```

Comment:
One thing that I didn't have clear is if we want this files to be generated. In my opinion, we should generate this documentation in json version and include it into the pug files. The bad thing is that that approach will take me some more time, so I guess this is Ok for the moment.